Marlin/src/pins/stm32f4/pins_BTT_BTT002_V1_0.h

```diff
@@ -225,20 +225,49 @@
 
     #define LCD_SDSS                        PA4
 
-    #define LCD_PINS_ENABLE                 PE9
-    #define LCD_PINS_D4                     PE10
+    #if ENABLED(FYSETC_MINI_12864)
+      #define DOGLCD_CS                     PE9
+      #define DOGLCD_A0                     PE8
+      #define DOGLCD_SCK                    PA5
+      #define DOGLCD_MOSI                   PA7
+
+      #define LCD_BACKLIGHT_PIN             -1
+
+      //#define FORCE_SOFT_SPI                      // Use this if default of hardware SPI causes display problems
+                                                  //   results in LCD soft SPI mode 3, SD soft SPI mode 0
```

Comment:
This option must be disabled in order for the LCD to work at all. With this enabled, the SPI calls are all no-op.

Reply:
After adjusting the rest of the pins file and making it a little easier to follow. FORCE_SOFT_SPI now functions and the everything is now working.